frontend/uddhar-frontend/src/authentication/pages/ForgetPass.jsx

```
@@ -0,0 +1,98 @@
+                    </h2>
+                    <input
+                        className="bg-transparent px-2 py-2 w-full mb-4 border-b-2 border-gray-300 focus:outline-none focus:border-amber-700"
+                        type="email"
+                        placeholder="Enter your email"
+                        value={email}
+                        onChange={(e) => setEmail(e.target.value)}
+                    />
+                    <button
+                        onClick={handleSendCode}
+                        className="w-full rounded-md bg-amber-700 text-white text-sm md:text-xl py-2"
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)